prettier.novaextension/extension.json

```diff
@@ -54,204 +58,241 @@
 					"link": "https://prettier.io/docs/en/options.html#prose-wrap",
 					"type": "enum",
+					"radio": false,
 					"default": "preserve",
 					"values": ["always", "never", "preserve"]
 				},
 				{
 					"key": "prettier.default-config.htmlWhitespaceSensitivity",
-					"title": "HTML whitespace sensitivity",
-					"description": "Specify the global whitespace sensitivity for HTML files, see Prettier documentation formatting for more info.",
+					"title": "HTML Whitespace Sensitivity",
+					"description": "Specify the global whitespace sensitivity for HTML, Vue, Angular, and Handlebars. See Prettier documentation for more info.",
 					"link": "https://prettier.io/docs/en/options.html#html-whitespace-sensitivity",
 					"type": "enum",
+					"radio": false,
 					"default": "css",
 					"values": ["css", "strict", "ignore"]
 				},
 				{
 					"key": "prettier.default-config.vueIndentScriptAndStyle",
-					"title": "Ident vue file script and style tags",
-					"description": "Whether or not to indent the code inside <script> and <style> tags in Vue files. Some people (like the creator of Vue) don’t indent to save an indentation level, but this might break code folding in your editor.",
-					"link": "https://prettier.io/docs/en/options.html#vue-indent-script-and-style",
+					"title": "Vue files script and style tags indentation",
+					"description": "Whether or not to indent the code inside <script> and <style> tags in Vue files.",
+					"link": "https://prettier.io/docs/en/options.html#vue-files-script-and-style-tags-indentation",
 					"type": "boolean",
 					"default": false
 				},
 				{
 					"key": "prettier.default-config.endOfLine",
-					"title": "End of line",
-					"description": "",
+					"title": "End of Line",
+					"description": "Newline control character",
 					"link": "https://prettier.io/docs/en/options.html#end-of-line",
 					"type": "enum",
+					"radio": false,
 					"default": "lf",
 					"values": ["lf", "crlf", "cr", "auto"]
+				},
+				{
+					"key": "prettier.default-config.embeddedLanguageFormatting",
+					"title": "Embedded Language Formatting",
+					"description": "Control whether Prettier formats quoted code embedded in the file.",
+					"link": "https://prettier.io/docs/en/options.html#embedded-language-formatting",
+					"type": "enum",
+					"radio": false,
+					"default": "auto",
+					"values": ["auto", "off"]
+				},
+				{
+					"key": "prettier.default-config.singleAttributePerLine",
+					"title": "Single Attribute Per Line",
+					"description": "Enforce single attribute per line in HTML, Vue, and JSX.",
+					"link": "https://prettier.io/docs/en/options.html#single-attribute-per-line",
+					"type": "boolean",
+					"default": false
 				}
 			]
 		},
 		{
 			"key": "prettier.format-on-save.section",
-			"title": "Format on save",
-			"description": "Can be overridden in project settings (except for ignored syntaxes).",
+			"title": "Format on Save Settings",
```

Comment:
We're already in the settings pane, so no need for the 'Settings' at the end I'd say